app/javascript/components/project-submissions/components/submissions-list.jsx

```diff
@@ -51,12 +51,14 @@ const SubmissionsList = ({
               />
             ))}
           </FlipMove>
+        ) : (
+          <h2 className="text-center text-gray-600 font-bold pt-6 pr-0 pb-2 mb-0">No Submissions yet, be the first!</h2>
```

Comment:
The text here looks a little small, could we make it bigger with `text-xl`? and can we also include a little more top padding `pt-14` gives it a bit more space to stand out imo.